pkg/container/restore.go

```diff
@@ -141,7 +140,7 @@ func containerdRestore(id string, ref string) error {
 	return nil
 }
 
-func RuncRestore(imgPath string, containerId string, criuOpts *CriuOpts, opts *RuncOpts) error {
+func RuncRestore(imgPath string, criuOpts *CriuOpts, opts *RuncOpts) error {
 	var spec rspec.Spec
 
 	configPath := opts.Bundle + "/config.json"
```

Comment:
This portion is needed bc this containerID is different than the containerID from opts. This containerID is the sleeping containerID bundle path. Sorry this wasn't commented or made more clear.

When we have the custom shim w/ c/r, we will no longer need this.